src/Faker/Provider/es_PE/Person.php

```diff
@@ -33,7 +33,8 @@ class Person extends \Faker\Provider\Person
         'Juan Pablo', 'Juan Sebastián', 'Julián', 'Kevin', 'Lautaro', 'Leonardo', 'Lorenzo', 'Lucas', 'Luciano', 'Luis', 'Manuel', 'Mario',
         'Martín', 'Mateo', 'Matthew', 'Matías', 'Maximiliano', 'Miguel', 'Miguel Ángel', 'Máximo', 'Nahuel', 'Pablo', 'Pedro', 'Rafael',
         'Ricardo', 'Rodrigo', 'Samuel', 'Santiago', 'Santino', 'Sebastián', 'Sergio', 'Simón', 'Thiago', 'Tomas', 'Valentino', 'Valentín',
-        'Vicente', 'Bautista', 'Hidalgo', 'Juan Esteban',
+        'Vicente', 'Bautista', 'Hidalgo', 'Juan Esteban', 'Jaime', 'Gerardo', 'Raúl', 'Emmanuel', 'Bernabé', 'Moisés', 'Abraham', 'Noé',
```

Comment:
please revert this and the following:

- names should be ordered alphabetically
- you should provide an open-source list of names as reference for your change.